connectors/sources/sharepoint_online.py

```diff
@@ -1775,3 +1788,13 @@ async def close(self):
 
     def advanced_rules_validators(self):
         return [SharepointOnlineAdvancedRulesValidator()]
+
+    def is_supported_format(self, filename):
+        if "." not in filename:
+            return False
+
+        attachment_extension = list(os.path.splitext(filename))
```

Comment:
nit: I think `list` isn't required, because you can use the following `[-1]` on the tuple result.